torch/csrc/jit/codegen/cuda/shape_expr_memo.cpp

```diff
@@ -0,0 +1,123 @@
+#include <torch/csrc/jit/codegen/cuda/shape_expr_memo.h>
+
+#include <torch/csrc/jit/codegen/cuda/glfdc/expr_builder.h>
+
+namespace fuser_cuda = torch::jit::fuser::cuda;
+
+using namespace fuser_cuda::kir;
+
+using fuser_cuda::glfdc::ExprDAG;
+using fuser_cuda::glfdc::ExprEvaluator;
+using fuser_cuda::glfdc::Operand;
+using fuser_cuda::glfdc::Value;
```

Comment:
These `using` seem to make the code bug prone. For example, `Val` is `kir::Val`, but `Value` is `glfdc::Value`, and that seems too subtle to notice and remember. And since this is in the fuser namespace, `Val` may actually mean the fuser `Val`. For safety and clarity, I'd suggest the `kir` and `glfdc` prefix should not be dropped except it's extremely clear.